featuretools/tests/synthesis/test_deep_feature_synthesis.py

```diff
@@ -162,6 +162,18 @@ def test_ignores_variables(es):
         assert 'value' not in variables
 
 
+@pytest.fixture
```

Comment:
This shouldn't be a pytest fixture.  Fixtures are functions that get used by the test cases.  For example, this test case uses the `es` fixture, which returns an entityset.